### ISSUE TITLE
settings tab remains empty (V 1.0.3)

### ISSUE THREAD

**Issue by notspecified** [derqurps/noefinderlein#4]
the settings tab remains blank in android N, see the screenshot:
![screenshot_20160418-180146](https://cloud.githubusercontent.com/assets/5119558/14611512/ec412320-0592-11e6-8ce2-2fb7cfac42aa.png)
with 5.1.1 everything works fine and im able to select the year.

**Comment by derqurps** [comment]
Is this only on tablet? I don't support tablets in this app because of too less time for programming. I also don't have a tablet, and the emulator is very slow for debugging.
I will look into this issue on mobile phones with N the next time I find time.

**Comment by notspecified** [comment]
yes, its only on the tablet! i also don't know if its related to the hardware, the resolution or to the android version. would it be useful if i make a log with logcat or smoething similar?

**Comment by derqurps** [comment]
on N on an emulated phone it does work...maybe an error in the preview release version of N for your tablet?

**Comment by notspecified** [comment]
possible! but how to figure that out? would a logfile with logcat help to find the problem?

**Comment by derqurps** [comment]
if you can provide that, it would clearly help

**Comment by notspecified** [comment]
damn! I'm not rooted on this tablet and logcat needs root to read the actions of other programs...
do you know a way how to acess a logfile?

some usb debugging stuff via adb?

**Comment by derqurps** [comment]
i'm sorry, i am also not very good with adb...
I only debug in android studio

**Comment by notspecified** [comment]
i'll try it this weekend, perhaps i get it working

**Comment by derqurps** [comment]
@notspecified is this issue resolved?

**Comment by notspecified** [comment]
It depends: On my Tablet (Pixel C, Android 7.1.2) everything is the same, just like one year ago.
On my SGS2 Android 7.1.1 (lineageOS) is everything fine, and the 3 options (choose yar manually, chose year und offlinemode) are available.
